### DISCUSSION TITLE
Transferring Passkey from one Device to another

### DISCUSSION THREAD

**Discussion by Absaar1548** [MasterKale/SimpleWebAuthn#419] [General]
"I am working on a passkey demo and have successfully implemented a basic registration and authentication flow. Now, I want to add a feature that allows the transfer of a passkey, mainly a private key, stored on Device 1 to Device 2. Is there a way to extract the private key from one device and load it onto another device?"

**Comment by lmarschall** [comment]
As far as I know, you won't be able to retrieve the specific key directly from the authenticator on your device, but passkeys are synced across your iCloud / Google Keychain eventually.
